src/codecs/resize/options.tsx

```diff
@@ -135,7 +135,7 @@ export default class ResizerOptions extends Component<Props, State> {
                 onChange={this.onChange}
               >
                 <option value="stretch">Stretch</option>
-                <option value="cover">Cover</option>
+                <option value="contain">Contain</option>
```

Comment:
I have no idea why this was called "cover" before. It's clearly "contain", and other bits of code referred to it as "contain".